src/VM/OpcodeFactory.cpp

```diff
@@ -200,189 +201,190 @@
+        case 0x810E: return make_unique<Opcode810EHandler>(vm);
+        case 0x810F: return make_unique<Opcode810FHandler>(vm);
+        case 0x8113: return make_unique<Opcode8113Handler>(vm);
+        case 0x8115: return make_unique<Opcode8115Handler>(vm);
+        case 0x8116: return make_unique<Opcode8115Handler>(vm);
+        case 0x8117: return make_unique<Opcode8117Handler>(vm);
+        case 0x8118: return make_unique<Opcode8118Handler>(vm);
+        case 0x8119: return make_unique<Opcode8119Handler>(vm);
+        case 0x811C: return make_unique<Opcode811CHandler>(vm);
+        case 0x811D: return make_unique<Opcode811DHandler>(vm);
+        case 0x811E: return make_unique<Opcode811EHandler>(vm);
+        case 0x8120: return make_unique<Opcode8120Handler>(vm);
+        case 0x8121: return make_unique<Opcode8121Handler>(vm);
+        case 0x8122: return make_unique<Opcode8122Handler>(vm);
+        case 0x8123: return make_unique<Opcode8123Handler>(vm);
+        case 0x8126: return make_unique<Opcode8126Handler>(vm);
+        case 0x8127: return make_unique<Opcode8127Handler>(vm);
+        case 0x8128: return make_unique<Opcode8128Handler>(vm);
+        case 0x8129: return make_unique<Opcode8129Handler>(vm);
+        case 0x812D: return make_unique<Opcode812DHandler>(vm);
+        case 0x812E: return make_unique<Opcode812EHandler>(vm);
+        case 0x812F: return make_unique<Opcode812FHandler>(vm);
+        case 0x8130: return make_unique<Opcode8130Handler>(vm);
+        case 0x8131: return make_unique<Opcode8131Handler>(vm);
+        case 0x8132: return make_unique<Opcode8132Handler>(vm);
+        case 0x8134: return make_unique<Opcode8134Handler>(vm);
+        case 0x8136: return make_unique<Opcode8136Handler>(vm);
+        case 0x8137: return make_unique<Opcode8137Handler>(vm);
+        case 0x8138: return make_unique<Opcode8138Handler>(vm);
+        case 0x8139: return make_unique<Opcode8139Handler>(vm);
+        case 0x813C: return make_unique<Opcode813CHandler>(vm);
+        case 0x8143: return make_unique<Opcode8143Handler>(vm);
+        case 0x8145: return make_unique<Opcode8145Handler>(vm);
+        case 0x8147: return make_unique<Opcode8147Handler>(vm);
+        case 0x8149: return make_unique<Opcode8149Handler>(vm);
+        case 0x814B: return make_unique<Opcode814BHandler>(vm);
+        case 0x814C: return make_unique<Opcode814CHandler>(vm);
+        case 0x814E: return make_unique<Opcode814EHandler>(vm);
+        case 0x8150: return make_unique<Opcode8150Handler>(vm);
+        case 0x8151: return make_unique<Opcode8151Handler>(vm);
+        case 0x8152: return make_unique<Opcode8152Handler>(vm);
+        case 0x8153: return make_unique<Opcode8153Handler>(vm);
+        case 0x8154: return make_unique<Opcode8154Handler>(vm);
+        case 0x9001: return make_unique<Opcode9001Handler>(vm);
+        case 0xC001: return make_unique<OpcodeC001Handler>(vm);
+        case 0xA001: return make_unique<OpcodeA001Handler>(vm);
```

Comment:
I guess I'm too accustomed to dynamic languages (PHP, JS), where I could just put class name in a string and instantiate class from it :)